docs/development/creating-editing-renovate-presets.md

```diff
@@ -0,0 +1,38 @@
+# Creating/editing Renovate presets
+
+Renovate comes with default presets that you can find in the `lib/config/presets/internal` directory.
+You can suggest changes to the presets with a pull request.
+
+Follow the rules below to increase the chance that your pull request gets merged.
+
+## General rules
+
+1. Only create/edit presets for problems which can not be fixed upstream
+1. The internal preset must be helpful for most Renovate users
+
+### Specific rules
+
+#### Group presets
+
+File location: `lib/config/presets/internal/group.ts`
+
+Rules:
+
+1. Only group dependencies that _must_ be updated together
```

Comment:
Actually we are not this strict.

We have the `group:monorepo` preset, although that extends from the `monorepo` one so the rules for that can be stated there.

The `group:recommended` preset is for related dependencies which aren't from a monorepo but which usually need to be updated together as separate PRs may each break.

Finally, any other `group:*` presets can be added here if they are beneficial to a wide number of users. They don't need to be added to `group:recommended`, meaning that users will "opt in" to them one by one and not get them automatically from `config:base`, which includes `group:monorepo` and `group:recommended`.

Reply:
I've copy/pasted your response into the document now. 😉